test/cypress/integration/social-share.test.js

```diff
@@ -0,0 +1,52 @@
+const socialShareButton = '[data-test="socialShareButton"]'
+const socialShareLinks = '[data-test="socialShareLinks"]'
+const socialShareSnackbar = '[data-test="socialShareSnackbar"]'
+
+describe('Social share', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('Social share expands on click and collapse on click.', () => {
+    cy.get(socialShareButton).click()
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('be.visible')
+    })
+    cy.get(socialShareButton).click()
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('not.be.visible')
+    })
+  })
+
+  it('Social share expands on enter and collapse on enter.', () => {
+    cy.get(socialShareButton).focus()
+    cy.get(socialShareButton).trigger('click')
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('be.visible')
+    })
+    cy.get(socialShareButton).trigger('click')
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('not.be.visible')
+    })
+  })
+
+  it('Social share links should collapse when a link is clicked.', () => {
+    cy.get(socialShareButton).click()
+    cy.get(`${socialShareButton}:first-child`).click()
+    cy.get(socialShareButton).children().each(link => {
```

Comment:
Think this should be `cy.get(socialShareLinks).children()` instead of `cy.get(socialShareButton).children()`. The `socialShareButton` doesn't have links, so they're not visible and the test passes.